release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -680,6 +680,11 @@ The node disruption policies feature has been promoted to GA. A node disruption
 [id="ocp-4-17-machine-management_{context}"]
 === Machine management
 
+[id="ocp-4-17-enhancements-aws-mapi-placement_{context}"]
+==== Supporting AWS Placement Group Partition Number
+
+This release introduces the `placementGroupPartition` field for {product-title} `MachineSet` on AWS. This feature allows users to specify a partition number within an existing placement group, enabling precise instance allocation and improved fault tolerance. For example, see xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machineset-aws-existing-placement-group_creating-machineset-aws[Assigning machines to placement groups for Elastic Fabric Adapter instances by using machine sets].
```

Comment:
Consider using the attribute for AWS: 

![Screenshot from 2024-10-03 13-37-19](https://github.com/user-attachments/assets/6076bd59-bf24-4abf-b880-fccc81a68130)